### PR TITLE
Change the subscriptionId paremeter to be completely optional

### PR DESCRIPTION
**Pull request by harvey-k**
The Azure Login action already sets your subscription id:
https://github.com/Azure/login/blob/master/src/main.ts#L39

This change makes usage a bit cleaner.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This action can be used to deploy Azure Resource Manager templates at different
 
 * `scope`: Provide the scope of the deployment. Valid values are: `resourcegroup`(default) , `subscription`, `managementgroup`. 
 * `resourceGroupName`: **Conditional** Provide the name of a resource group. Only required for Resource Group Scope
-* `subscriptionId`: **Conditional** Provide the subscription ID  which should be used. Only required for scope `resourcegroup` & `subscription`. 
+* `subscriptionId`: **Conditional** Provide a value to override the subscription ID set by [Azure Login](https://github.com/Azure/login).
```
Comment (bishal-pdMSFT) [inline README.md:16]: @pulkitaggarwl please review this

File: src/main.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@ export async function main(): Promise<Outputs> {
     const managementGroupId = getInput('managementGroupId')
 
     // change the subscription context
-    if (scope != "managementgroup") {
+    if (subscriptionId !== "") {
```
Comment (bishal-pdMSFT) [inline src/main.ts:27]: I think we should still have the check for `scope` as subscription should get ignored for managementGroup scope

**Review by bishal-pdMSFT — COMMENTED** [review]
Left couple of comments, otherwise looks good to me.

File: src/main.ts (hunk 2 above)
Comment (harvey-k) [reply src/main.ts]: Ok, e08dd59 should fix this.

**Comment by bishal-pdMSFT** [comment]
So sorry 😞, this went completely under the radar. 

The change looks good to me. @pulkitaggarwl / @kanika1894 any comments? Pulkit, please look at [this](https://github.com/Azure/arm-deploy/pull/19/files#r501444386) as well.

**Comment by kanika1894** [comment]
> So sorry 😞, this went completely under the radar.
> 
> The change looks good to me. @pulkitaggarwl / @kanika1894 any comments? Pulkit, please look at [this](https://github.com/Azure/arm-deploy/pull/19/files#r501444386) as well.

looks good to me.

**Comment by pulkitaggarwl** [comment]
> > So sorry 😞, this went completely under the radar.
> > The change looks good to me. @pulkitaggarwl / @kanika1894 any comments? Pulkit, please look at [this](https://github.com/Azure/arm-deploy/pull/19/files#r501444386) as well.
> 
> looks good to me.

Looks good. Thanks @harvey-k  for the contribution

**Pull request merged**